.travis.yml

```diff
@@ -0,0 +1,24 @@
+language: go
+
+services: docker
+jobs:
+    include:
+        - stage: Debricked-scan
+          on:
+            branch: "*"
+          env:
+          -  DEBRICKED_TOKEN=${DEBRICKED_TOKEN}
+
+          before_install: 
+          -  env > file.env  
+          -  printf "$(go mod graph)\n\n$(go list -mod=readonly -e -m all)" > .debricked-go-dependencies.txt
+
+          cache:
+            directories:
+              - $HOME/.cache/go-build
+              - $HOME/gopath/pkg/mod
```

Comment:
After reading up a bit this look like a good option! 
Have you verified that go modules are indeed stored there in the builds?